index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="ko">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="style.css">
+		<title>판다마켓</title>
+	</head>
+	<body>
+		<!-- header section / 로고, 로그인 -->
+		<header>
+			<div class="banner">
+				<div class="banner-main">
+					<a class="header-logo" href="index.html">
+						<img src="./images/header-logo.png" alt="로고">
+					</a>
+					<a id="login" href="login.html">로그인</a>
+				</div>
+			</div>
+		</header>
+		<!-- main section -->
+		<main>

```

Comment:
💊 제안

주석을 다는 것은 좋은 습관이지만, 가장 좋은 코드는 주석 없이도 이해할 수 있는 코드입니다.
현재 코드 구조상 주석 없이도 충분히 이해할 수 있어 보이는데, 오히려 주석이 추가적인 정보를 전달하지 못하고 가독성을 해치고 있는 것 같아요.
필요한 부분에만 주석을 추가하면 더 좋은 코드가 될 것 같습니다!

```suggestion
		<main>
```